### ISSUE TITLE
Release 0.11.1 import error

### ISSUE THREAD

**Issue by rodriguez-facundo** [pyecore/pyecore#73]
Hi 👋, we are seeing this error in 2 repositories. Reverting to 0.11.0 fix it.

![import error](https://user-images.githubusercontent.com/32278395/68049461-b6f66700-fcc1-11e9-9eea-9bb07aa4587c.png)

**Comment by aranega** [comment]
Thanks for the ticket, I'll check that asap. I didn't do so much modifications in `0.11.1`, so I probably did a very dumb mistake somewhere.

**Comment by aranega** [comment]
In the same time, could you try to change the import you did from `from pyecore.valuecontainer import EList` to `from pyecore.ecore import EList` and tell me if it fixes the issue? (I'm doing changes that should fix this circular reference)

**Comment by aranega** [comment]
Ok, I made some quick experiment and changing `from pyecore.valuecontainer import EList` to `from pyecore.ecore import EList` fixes the issue. I will try to find a way of fixing the circular import, it should not be an issue anyway.

**Comment by rodriguez-facundo** [comment]
Thanks, @aranega  💯

**Comment by aranega** [comment]
Did the modification in your code fix the issue?

**Comment by rodriguez-facundo** [comment]
Nop. I keep getting `cannot import ECollection` both with `.valuecontainer` and `pyecore.ecore`. I also tried `pyecore.valuecontainer`

**Comment by aranega** [comment]
I reproduced your issue with `org.geppetto.frontend.jupyter`, I installed all the dependencies, running `pyecore` in version `0.11.1` and just creating a file with this content:

```python
from jupyter_geppetto.websocket_connection import TornadoGeppettoWebSocketHandler
```

However, when I change `from pyecore.valuecontainer import EList` to `from pyecore.ecore import EList` line 10 of the `websocket_connection.py` module, the problem disapears. Perhaps the issue comes again but from another line (another import of `ECollection` somewhere)? In my opinion, it would be preferable to change those imports in the `org.geppetto.frontend.jupyter` project and in `pygeppetto` if there is some, the `valuecontainer` module was not made to be exposed in the first place (but it should not cause problems anyway).

I will still try to fix the circular import on my side, but the last test I made show that it will be a little bit more complex than what I expected.
